css-canvas-medical/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Canvas Medical</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="header">
+          <div class="col-left-header">
+            <img src="images/logo-blue.svg" class="header-logo" />
+          </div>
+          <div class="col-right-header">
+            <div id="product" class="header-button">
+              <button>Product</button>
+            </div>
+            <div id="implementation" class="header-button">
+              <button>Implementation</button>
+            </div>
+            <div id="about-us" class="header-button">
+              <button>About Us</button>
+            </div>
+            <div id="blog" class="header-button"><button>Blog</button></div>
```

Comment:
Rather than making these `<button>`'s, I would've made them links (`<a>`). Since they don't look like a typical button, it's kind of confusing to make them a button